extensions/amp-experiment/0.1/test/test-amp-experiment.js

```diff
@@ -145,4 +145,13 @@ describes.realWin('amp-experiment', {
           .to.equal(null);
     });
   });
+
+  it('should access document from ampdoc', () => {
+    addConfigElement('script');
```

Comment:
I understand this is difficult to do unit test here. But I don't think this test is adding value other than making sure that line of code is not touched.

Sounds like calling `this.win.document` and `doc.body` is in general a dangerous thing, should we add a presubmit check? @choumx 

I suggest you revert this test case, and we can have that check in a separate PR.

Reply:
A presubmit check for code in `extensions/` might be a good idea.

Reply:
reverted

Reply:
@choumx Should @prateekbh add this presubmit check or would you prefer it in a separate PR?

Reply:
Separately is fine. Filed #19514.

Reply:
We should have a PWA integration test for each AMP component, including this one.